modules/fedimint-ln-client/src/cli.rs

```diff
@@ -43,6 +51,35 @@ enum Opts {
         #[clap(long, default_value = "false")]
         force_internal: bool,
     },
+    /// Register and manage LNURLs
+    #[clap(subcommand)]
+    Lnurl(LnurlCommands),
+}
+
+#[derive(Subcommand, Serialize)]
+enum LnurlCommands {
+    /// Register a new LNURL payment code with a specific LNURL server
+    Register {
+        /// The LNURL server to register with
+        server_url: SafeUrl,
+        /// Set LNURL meta data, see LUD-06 for more details on the format
+        #[clap(long)]
+        meta: Option<String>,
+        ///Shrthand for setting the short description in the LNURL meta data
+        #[clap(long, default_value = "Fedimint LNURL Pay")]
+        description: String,
+    },
+    /// List all LNURLs registered
+    List,
+    /// List all invoices and their operation ids generated for a LNURL
+    Invoices { payment_code_idx: u64 },
+    /// List details for a specific invoice by operation id
+    InvoiceDetails { operation_id: OperationId },
```

Comment:
I don't see this tested in the devimint test, lets add it just to make sure it doesn't break